internal/test/e2e/run.go

```diff
@@ -321,16 +322,21 @@ func splitTests(testsList []string, conf ParallelRunConf) ([]instanceRunConf, er
 	}
 
 	if strings.EqualFold(conf.BranchName, conf.BaremetalBranchName) {
-		runConfs, err = splitTinkerbellTests(awsSession, testsList, conf, testRunnerConfig, runConfs)
+		runConfs, err = splitNonAirgapTinkerbellTests(awsSession, testsList, conf, testRunnerConfig, runConfs)
```

Comment:
Confused by the name and method: we generate the non airgap test runConfs first, then append the airgap test runConfigs after?

This is kinda hard to review. Can we decouple the 2 and write methods that only deal w each type of testing? e.g.

```go
func airgapRunConfs(airgapTests []string) []runConf 

func nonAirgapRunConfs(nonAirgapTests []string) []runConf  
```

Reply:
I have renamed the two functions to make them easier to reason about. I think a more serious refactor would be nice, but it's pretty hard to include the refactoring in this PR.